packages/devextreme/js/__internal/ui/chat/messagebubble.ts

```diff
@@ -21,8 +22,13 @@ class MessageBubble extends Widget<Properties> {
   }
 
   _initMarkup(): void {
-    $(this.element())
-      .addClass(CHAT_MESSAGEBUBBLE_CLASS);
+    const $element = $(this.element());
+
+    $element.addClass(CHAT_MESSAGEBUBBLE_CLASS);
+
+    $('<div>')
```

Comment:
Should we move the styles from `dx-messagebubble` and apply them to the new content element? I mean, now we don't have direct access to the container to add an additional class to it and override styles if needed,   so, what element should have styles `dx-messagebubble` or `dx-messagebubble-content`?

Reply:
discussed in voice, since we want to have a possibility to add bubbleHeader or bubbleFooter later, we should leave styles on root element, so they would be applied to header/footer